### DISCUSSION TITLE
Chatting about Dev Generally

### DISCUSSION THREAD

**Discussion by BadgerOnABike** [PSaaS-Developers/Project_issues#65] [General]
### Good news: 
- we got modelling running with PSaaS v7 on Linux.
- a few basic tests that I did show the same output between v6 and v7.
- other than these tweaks and a few fixes, it's the build I gave you bad week.
- we've sped up generation of FGMJ files from 2 minutes to 12 seconds for Cdn weather streams, thanks to some tweaks Travis was able to do.
- NZ wants PSaaS as part of a demo or presentation at a workshop on May 5(ish).
### Bad news: 
- we missed some pulls of functionality from v6 to v7 w/r to the Geoserver work.
- similar to the hotfix last year, I need to do the same thing in v6 on this build.
- I want to fix these builds for you right away, particularly if nobody has used them yet.

W/R to the bad news, I now realize that this is the first real push for using V7 with sub-scenarios AND with Geoserver, too.  So I guess I can't be surprised about these few hiccups, but it came together quickly.  And you have to tell me how you could accommodate quick fixes like this.

**Comment by spydmobile** [comment]
Ok, so I have created a Mini Project to handle the 3 bugs #66 #67 #68 (@RobBryce Created) and grouped them together as:
https://github.com/PSaaS-Developers/Project_issues/projects/1

**Reply by RobBryce** [reply]
Thanks, working on that now.
